<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">SimpleConvert</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/icons/simpleconvert</normaloff>:/icons/simpleconvert</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="hLayoutCentralWidget">
      <item>
       <layout class="QVBoxLayout" name="vLayoutMain">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QListWidget" name="lvFiles">
          <property name="selectionMode">
           <enum>QAbstractItemView::ExtendedSelection</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblOutputExtension">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Output extension:</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbOutputType">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string notr="true">mp3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">flac</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string notr="true">mp4</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblOutputFolder">
          <property name="text">
           <string>Output Folder:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="hLayoutOutputFolder">
          <property name="spacing">
           <number>5</number>
          </property>
          <item>
           <widget class="QLineEdit" name="editOutputFolder">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSelectOutputFolder">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Browse</string>
            </property>
            <property name="icon">
             <iconset>
              <normalon>:/icons/folder</normalon>
             </iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="vLayoutMainButtons">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnAddFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/icon_add</normalon>
           </iconset>
          </property>
          <property name="shortcut">
           <string notr="true" extracomment="Add Files">Ctrl+N</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveFile">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string notr="true"/>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/icons/icon_remove</normalon>
           </iconset>
          </property>
          <property name="shortcut">
           <string notr="true" extracomment="Remove File"/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="hLayoutStatusBar">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="QLabel" name="lblStatus">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="pbStatus">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblLoadingIndicator">
        <property name="text">
         <string notr="true">LOADING</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>He&amp;lp</string>
    </property>
    <addaction name="actionAbout_SimpleConvert"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConvert_Files"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionPreferences"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout_SimpleConvert">
   <property name="text">
    <string>&amp;About SimpleConvert</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionConvert_Files">
   <property name="text">
    <string>&amp;Convert Files</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Return</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
